lib/pacemaker/pcmk_sched_probes.c

```diff
@@ -482,10 +484,9 @@ add_start_orderings_for_probe(pe_action_t *probe, pe_action_wrapper_t *after)
         return;
     }
 
-    crm_trace("Adding probe start orderings for '%s@%s (%s) "
+    crm_trace("Adding probe start orderings for 'unrunnable %s@%s "
               "then instances of %s@%s'",
               probe->uuid, pe__node_name(probe->node),
-              pcmk_is_set(probe->flags, pe_action_runnable)? "runnable" : "unrunnable",
```

Comment:
Can `pe_action_runnable` not be unset? We drop the check in this commit, here and below. I'd rather not trace all the way through for a trace message if you've already done so.

Reply:
The statement before this returns if probe is runnable :)

Reply:
Lol whoops